app/src/main.ts

```diff
@@ -75,7 +75,7 @@ async function init(socket: WebSocket) {
 
   if (app === 'webview') {
     const webview = Deno.run({
-      cwd: dirname(new URL(Deno.mainModule).pathname),
+      cwd: dirname(fromFileUrl(new URL(Deno.mainModule))),
```

Comment:
I think you should be able to do just `dirname(fromFileUrl(Deno.mainModule))` here, no need to do the `new URL()`. I can't test this on windows, though.

Otherwise looks great!

Reply:
Thanks again.
Yeah this seems to build and run as expected on my Windows machine. I force pushed over it as well.